Controls/main.c

```diff
@@ -0,0 +1,17 @@
+#include "MotorController.h"
+#include "stdlib.h"
+#include <time.h>
+
+void delay(int mseconds){
+    clock_t start_time = clock();
+    while (clock()<start_time+mseconds){
+        ;
+    }
+}
+
+void main(){
+    while(1==1){
```

Comment:
A while loop can just be written as `while (1)` in C. Any number besides 0 evaluates to true